src/test/java/edu/ucla/library/iiif/auth/services/AccessCookieServiceTest.java

```diff
@@ -34,11 +37,13 @@
 import io.vertx.junit5.VertxTestContext;
 import io.vertx.serviceproxy.ServiceBinder;
 import io.vertx.serviceproxy.ServiceException;
+import io.vertx.sqlclient.Tuple;
 
 /**
  * Tests the {@link AccessCookieService}.
  */
 @ExtendWith(VertxExtension.class)
+@TestInstance(Lifecycle.PER_CLASS)
```

Comment:
Interesting... I'm curious what the value is to using this vs. making the BeforeAll and AfterAll static(?)

Reply:
In this case since there are member variables, I didn't see the latter as a viable route

Reply:
They could have been static too since they're only set once before all the tests? Either way, was just curious... thanks.

Reply:
I don't see how they could be since they depend on the result of a Future (config)

Reply:
FTR, I stand corrected, they could indeed be static. Thanks for the tip

Reply:
If the fields and the setup and teardown methods are all static (and the fields aren't final), it should work. But, I think I may like it better this way anyway.